components/brave_rewards/resources/shared/components/unsupported_region_notice/unsupported_region_notice.tsx

```diff
@@ -0,0 +1,100 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import * as React from 'react'
+
+import { LocaleContext, formatMessage } from '../../lib/locale_context'
+import { NewTabLink } from '../new_tab_link'
+import { BatCrossedIcon } from '../icons/bat_crossed_icon'
+
+import * as style from './unsupported_region_notice.style'
+
+interface Props {
+  ntpCard: boolean
+}
+
+export function UnsupportedRegionNotice (props: Props) {
```

Comment:
It looks like maybe this component should be split into two components (one for the NTP, and one for the Rewards page) since the styles are different and just the strings are shared?

If we split the component and just put them into the folders where they are needed, then it looks like maybe `BatCrossedIcon` doesn't need to be shared either.

Reply:
Fixed via 9f111ed50548b90b6c797a561ef0fec5f119a8ea. PTAL.

Reply:
Looks great.